deckhand/engine/schema/v1_0/default_schema.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2017 AT&T Intellectual Property.  All other rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+substitution_schema = {
```

Comment:
Well, according to the API docs [0], the jsonchema constructor expects a dictionary as the schema to validate the data against. So the one issue with using a JSON file as the schema is that it would require opening it and reading it, then calling JSON.parse() on it. This is more error-prone than simply importing the schema directly. I personally prefer including the schema in Python code -- which is what Tempest also does (which means there's precedence for it).

[0] https://python-jsonschema.readthedocs.io/en/latest/validate/#jsonschema.IValidator